src/t2iapi/combined/combined_requests.proto

```diff
@@ -0,0 +1,28 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2023 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.combined;
+
+import "t2iapi/context/types.proto";
+import "t2iapi/operation/types.proto";
+
+option java_package = "com.draeger.medical.t2iapi.combined";
+option java_outer_classname = "CombinedRequests";
+
+
+/*
+Request to create a new or adapt an existing context state with a requested @ContextAssociation and
+set the requested @OperatingMode for the referenced operation.
+ */
+message CreateContextStateWithAssociationAndSetOperatingModeRequest {
```

Comment:
would suggest describing other parmeters as well, even if they might be self-explanatory at first glance

Reply:
Added descriptions to the parameter